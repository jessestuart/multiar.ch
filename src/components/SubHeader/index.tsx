import React from 'react'
import { Text } from 'rebass'

const SubHeader = () => {
  return (
    <div className="pv4 bb bt b--black-10 flex justify-center">
      <Text className="flex flex-column lh-title mw7">
        <Text
          fontFamily="serif"
          className="i justify-center center pl3"
          style={{
            borderLeft: '2px rgba(104, 70, 91, 0.7) solid',
          }}
        >
          Cross-build so hard, muhfuckas wanna fine me;
          <br />
          (that manifest cray)
        </Text>
        <div className="flex justify-end">
          <Text fontFamily="serif" pl={2}>
            â€” Jay-Z, Probably
          </Text>
        </div>
      </Text>
    </div>
  )
}

export default SubHeader
