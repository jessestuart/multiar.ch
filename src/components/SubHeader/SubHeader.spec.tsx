import React from 'react'
import renderer from 'react-test-renderer'

import SubHeader from 'components/SubHeader'

test('SubHeader component.', () => {
  const component = renderer.create(<SubHeader />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
