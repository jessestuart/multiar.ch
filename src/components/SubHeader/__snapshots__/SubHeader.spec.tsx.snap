// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubHeader component. 1`] = `
.c2 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  font-family: Alegreya;
}

.c0 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c1 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div
  className="c0 pt5 pb4 bb bt b--black-10 flex justify-center pl4 pl0-ns"
>
  <div
    className="c1 lh-title"
  >
    <div
      className="c2 i justify-center center pl3"
      fontFamily="serif"
      style={
        Object {
          "borderLeft": "2px #68465b solid",
        }
      }
    >
      Cross-build so hard, muhfuckas wanna fine me;
      <br />
      (that manifest cray)
    </div>
    <div
      className="c0 justify-end"
    >
      <div
        className="c2"
        fontFamily="serif"
      >
        â€” Jay-Z, Probably
      </div>
    </div>
  </div>
</div>
`;
